docs/tutorials/python-api.md

```diff
@@ -23,14 +23,13 @@ Start JupyterLab:
 jupyter lab
 ```
 
-And, launch a new notebook.
+And launch a new notebook.
 
 ## Step 1: Setup RAG configuration
 
 The first step is to setup the configuration for components like the source storage
 (vector database), API, assistant (LLM), and more. You can set this using a
-`config.toml` file, learn more in the
-[how-to guide for setting configuration](../how-tos/set-configuration.md).
+`config.toml` file.
```

Comment:
Let's call it

```suggestion
`ragna.toml` file.
```

since that is the default name.

Reply:
[changed](https://github.com/Quansight/ragna/pull/196/commits/6f34a65d837f02aec9c8c430b12964ca3b779563) as requested